rasa_nlu/evaluate.py

```diff
@@ -125,9 +126,9 @@ def plot_confusion_matrix(cm,
         fig.savefig(out, bbox_inches='tight')
 
 
-def plot_histogram(hist_data: List[List[float]],
-                   out: Optional[Text] = None
-                   ) -> None:  # pragma: no cover
+def plot_histogram(hist_data,  # type: List[List[float]]
+                   out=None  # type: Optional[Text]
+                   ):  # pragma: no cover
```

Comment:
Revert this

Reply:
yes, please do that :P

Reply:
you can do it with the suggestion function and then commit them all together